projects/Amlogic/options

```diff
@@ -143,3 +143,11 @@
   # Space separated list is supported,
   # e.g. ADDITIONAL_PACKAGES="PACKAGE1 PACKAGE2"
     ADDITIONAL_PACKAGES="u-boot-tools-aml"
+
+  # build and install driver addons (yes / no)
+    DRIVER_ADDONS_SUPPORT="yes"
+
+  # driver addons to install:
+  # for a list of additinoal drivers see packages/linux-driver-addons
+  # Space separated list is supported,
+    DRIVER_ADDONS="crazycat hauppauge"
```

Comment:
We are building them as a LE repository addons. Update for these  addons is less frequent then LE updates. Also we can save about 9MB in install image.

Reply:
It has pros and cons. What happens if we bump the kernel?

Reply:
@afl1 that approach only works without hassle at AML as there are no kernel updates, every other project would fail without nasty workarounds at up/downgrade scenarios

Reply:
The only image that has size constraints is Generic (x86_64) where we need to be aware of older OE installs with 235MB boot partitions. All other build projects are a mile clear of that restriction and we should embed drivers into the image to avoid mismatched driver issues from occurring. Bandwidth is cheap. Human effort debugging driver issues is not.